Polyhedron/demo/Polyhedron/Plugins/PCA/Scene_edit_box_item.cpp

```diff
@@ -773,6 +773,7 @@ void Scene_edit_box_item::highlight()
 {
   d->ready_to_hl = true;
   Viewer_interface* viewer = dynamic_cast<Viewer_interface*>(*QGLViewer::QGLViewerPool().begin());
+  viewer->makeCurrent();
```

Comment:
How will it be implemented in our future multi-viewer world?

Reply:
makeCurrent() is a function from Qt, not qglViewer, so it will be the same as I intend to use QOpenGLWidget (which works for Android, unlike the old QGLWidget)

Reply:
I mean: if there are multiple viewer, `QGLViewer::QGLViewerPool()->begin()` may not be the right viewer! I think the API of that item will have to be changed so that the `Scene_edit_box_item` knows in which viewer it is used.

Reply:
Oh, that. I could use the lambda function version of the singleShot to send the viewer as argument for the slot `highlight()`.

Reply:
An object should not be usable in only one viewer, especially if it is not in ht emain one, as it can be destroyed any time. The solution I propose would be consistent with what you are saying. Adding an argument to `highlight()` would avoid having to cast from the pool, and the function that calls highlight casts from its `obj`argument, which is the affected viewer. So I think it is the best solution.